hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/OzoneRepair.java

```diff
@@ -61,4 +66,27 @@ public OzoneConfiguration getOzoneConf() {
   public static void main(String[] argv) throws Exception {
     new OzoneRepair().run(argv);
   }
+
+  @Override
+  public int execute(String[] argv) {
+    String currentUser = getSystemUserName();
+    String s = getConsoleReadLineWithFormat(currentUser);
+    boolean shouldProceed = Boolean.parseBoolean(s) || "y".equalsIgnoreCase(s);
```

Comment:
```suggestion
    boolean shouldProceed = "y".equalsIgnoreCase(s);
```

I think this is sufficient. I'm not sure we need to support people typing "true" into the prompt.

Reply:
yeah, 'y' should be good! updated.